addon/utils/dom.js

```diff
@@ -104,3 +107,8 @@ export function getDestinationElement(context) {
 
   return destinationElement;
 }
+
+export function findElemementByIdInShadowDom(context, id) {
+  const owner = getOwner(context);
+  return owner.rootElement.querySelector && owner.rootElement.querySelector(`[id="${id}"]`);
```

Comment:
Is `querySelector('#my-id')` not working within a Shadow DOM?

Reply:
Yes.